libs/oeth/shared/project.json

```diff
@@ -5,14 +5,6 @@
   "projectType": "library",
   "tags": [],
   "targets": {
-    "codegen-graphql": {
```

Comment:
why removing this target?

Reply:
I had to remake the folders and then copy in the stuff we cared about so this is just a miss on my part.